### ISSUE TITLE
To be able to search on TOTP field

### ISSUE THREAD

**Issue by Gustry** [keepassxreboot/keepassxc#3260]
[TIP]:  # ( Provide a general summary of the feature in the title above ^^ )
[TIP]:  # ( DO NOT include screenshots of your actual database! )

## Summary
[NOTE]: # ( Provide a brief overview of what the new feature is all about )

I would like to search all items which have a TOTP field set.

## Desired Behavior
[NOTE]: # ( Tell us how the new feature should work, be specific )

Maybe adding a new criteria for searching items like this?
![Sélection_001](https://user-images.githubusercontent.com/1609292/59336530-07632d00-8cff-11e9-94c2-3c9f016d6a64.png)

## Context
[NOTE]: # ( Why does this feature matter to you? What unique circumstances do you have? )

I have a few TOTP setup in Keepass but it seems convenient for me to search items like this.

**Comment by droidmonkey** [comment]
Just search for the attribute name (for now): ```attr:otp```

**Comment by Gustry** [comment]
Ah yes it works. Thanks.
Should I close the issue or should the documentation/UX be improved?

**Comment by droidmonkey** [comment]
It is good to add it to search option

**Comment by rwat1** [comment]
YES, please, I used to use "Tags" in KeePass for TOTP entries, but this REALLY would make my life easier, since all my TOTP is scattered across so many groups and I need to scan each QR code quickly.
